hail/python/hailtop/batch/docs/service.rst

```diff
@@ -175,6 +175,13 @@ is standard. If a job is scheduled on this machine, then the cost per core hour
     total cost for that job will be 7 minutes.
 
 
+Batch also supports private instances that are created for a single job that specify
+:meth:`.Job.machine_type`, set :meth:`.Job.preemptible` to False, or request 32 or
+64 cores based on the combined cpu and memory requests. A job is billed for the entire
+time the machine is running including the activation time for the instance. The costs
+are the same as above except the cost for the Local SSD is omitted.
```

Comment:
All our pools are preemptible. If I read this correctly, private instances must be non-preemptible. Why provide two configurable variables? It seems like `preemptible` is unnecessary and `machine_type` fully describes the request. Perhaps machine_type should be instead `def private_machine(machine_type: int)`?